src/stories/1-Button.stories.js

```diff
@@ -14,29 +17,29 @@ export const Default = () => <Button>Save article</Button>;
 export const Active = () => <Button active>Save article</Button>;
 
 export const Dropdown = () => (
-  <DefaultWithIcon>
+  <ButtonWithIcon>
```

Comment:
Maybe this button should be named sth like NarrowButtonWithIcon or LongButtonWithIcon ...so it becomes clear which one it is without knowing that the other one is called SquareButtonWithIcon